src/components/Scaler/index.js

```diff
@@ -53,6 +53,16 @@ class Scaler extends Component {
         }
       }
     });
+
+    if (
+      !isEqual(
+        prevProps.dataContext.xSubDomain,
+        this.props.dataContext.xSubDomain
+      )
+    ) {
+      // eslint-disable-next-line
```

Comment:
Which eslint rule are you disabling here?

Reply:
(and why?)

Reply:
it's the "Do not use setState in componentDidUpdate" rule, which is ok to do as long as you don't introduce an infinite loop

Reply:
Sorry, that was meant to be rhetorical. Please use:
`// eslint-disable-next-line whatever/rule-is-broken`